### PR TITLE
feat: keep the toolbar the same height as the keyboard to optimize the editing experience.

### PR DESCRIPTION
**Pull request by LucasXu0**
#### before

https://github.com/AppFlowy-IO/AppFlowy/assets/11863087/41f3ee66-712a-4bb6-a4ca-e8c222d0f747

#### after

https://github.com/AppFlowy-IO/AppFlowy/assets/11863087/6862afd1-47fc-4097-b129-ad2dfdc3e853

---

#### PR Checklist

- [ ] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/mobile_toolbar_item/mobile_blocks_menu.dart [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      children: items.map((item) {
+        final selection = editorState.selection;
+        if (selection == null) {
+          return const SizedBox.shrink();
+        }
+        bool isSelected = false;
+        if (item.isSelected != null) {
+          isSelected = item.isSelected!(editorState, selection);
+        } else {
+          isSelected = _isSelected(editorState, selection, item.blockType);
```
Comment (Xazin) [inline frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/mobile_toolbar_item/mobile_blocks_menu.dart:64]:
```suggestion
        isSelected = item.isSelected?.call(editorState, selection) ?? _isSelected(editorState, selection, item.blockType);
```

File: frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/mobile_toolbar_item/mobile_text_decoration_item.dart [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+  @override
+  Widget build(BuildContext context) {
+    final style = MobileToolbarTheme.of(context);
+
+    final bius = textDecorations.map((currentDecoration) {
+      // Check current decoration is active or not
+      final selection = widget.selection;
+      final nodes = widget.editorState.getNodesInSelection(selection);
+      final bool isSelected;
```
Comment (Xazin) [inline frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/mobile_toolbar_item/mobile_text_decoration_item.dart:71]: This reads a bit odd

**Review by Xazin — APPROVED** [review]

**Comment by LucasXu0** [comment]
Screenshot 2023-11-16 at 21 15 51

File: frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/mobile_toolbar_item/mobile_blocks_menu.dart (hunk 1 above)
Comment (LucasXu0) [reply frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/mobile_toolbar_item/mobile_blocks_menu.dart:64]: I prefer to keep the previous one; the suggested one is not as easily readable as the first one.

**Pull request merged**